modin/pandas/test/test_io.py

```diff
@@ -1729,6 +1752,18 @@ def test_read_parquet_s3_with_column_partitioning(self, engine):
         )
 
 
+# Leave this test apart from the test classes, which skip the default to pandas
+# warning check. We want to make sure we are NOT defaulting to pandas for a
+# path relative to user home.
+# TODO(https://github.com/modin-project/modin/issues/3655): Get rid of this
+# commment once we turn all default to pandas messages into errors.
+def test_read_parquet_relative_to_user_home(make_parquet_file):
```

Comment:
We should use `@pytest.mark.filterwarnings(default_to_pandas_ignore_string)` for HDK backend.

Reply:
I've embedded the check to same place where we check for `Engine.get() == "Python"`

Reply:
Hmm, that wasn't entirely correct, so I changed it again 🤣